backend/src/users/users.controller.ts

```diff
@@ -16,14 +16,23 @@ export class UsersController {
         await sendEmail(user.email, url)
     }
 
+    @Get()
+    async findOne(@Query() query){
```

Comment:
Please type annotate `query` parameter here.

Reply:
I don't I will be needing it anymore.